zanata-war/src/main/java/org/zanata/rest/editor/service/resource/UserResource.java

```diff
@@ -59,7 +59,7 @@
      *
      * @param username
      */
-    public User generateUser(String username);
+    public User generateUser(String username, boolean includeEmail);
```

Comment:
I know this is already there but this method and the other one is not REST methods. They can stay private in the implementation class